renku/core/session/session.py

```diff
@@ -353,3 +353,8 @@ def ssh_setup(existing_key: Optional[Path] = None, force: bool = False):
             """
         )
         f.write(content)
+
+    communication.warn(
+        "This command does not add any public SSH keys into your project. "
+        "Keys have to be added manually or by using the renku session start command with the --ssh flag."
```

Comment:
The convention for such messages is to put commands/options/flags inside single quotes to differentiate them with the rest of the message: `... by using the 'renku session start' command with the '--ssh' flag.`